### ISSUE TITLE
Pulverizer rendering crash

### ISSUE THREAD

**Issue by murilorodelli** [anegostudios/VintageStory-Issues#1166]
**Game Version:** v1.15.0-rc.3 (Unstable)14/07/2021
**Platform:** Windows
**Modded:** / Yes (new TechRabbit modpack)  

### Description
Pulverizer Crash.

### How to reproduce
1. Make a pulverizer
2. Attach a hopper with an "L" chute on the side of the pulverizer
3. Add ores of different kinds in the hopper to pulverize 
4. It crashes 

### Expected behavior
Not to crash

### Logs
```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.15.0-rc.3 (Unstable)14/07/2021 10:57:51: Critical error occurred
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Vintagestory.GameContent.Mechanics.PulverizerRenderer.UpdateLightAndTransformMatrix(Int32 index, Vec3f distToCamera, Single rotation, IMechanicalPowerRenderable dev)
   at Vintagestory.GameContent.Mechanics.MechBlockRenderer.UpdateCustomFloatBuffer()
   at Vintagestory.GameContent.Mechanics.PulverizerRenderer.OnRenderFrame(Single deltaTime, IShaderProgram prog)
   at Vintagestory.GameContent.Mechanics.MechNetworkRenderer.OnRenderFrame(Single deltaTime, EnumRenderStage stage)
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.MainRenderLoop(Single dt)
   at Vintagestory.Client.NoObf.ClientMain.MainGameLoop(Single deltaTime)
   at _vTO2p6mYoYgQLd0czvTimWMtPVn._kS2WnQE2DCEcrAUbLAkHuTaFlFA(Single )
   at _4SI8Umxp6qAHEBPFOERJtwQBSOp._KbtVbLBs8jwmgtNYHzC8WuXmmlG(Single )
   at _4SI8Umxp6qAHEBPFOERJtwQBSOp._9HtfDIaMIPYmAqpIaQT4w9Q0Jni(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _7zCnJVmxgV4BmwWUrLLXlbJoJGn._wGFDpKw5npWpK2EoZcBqD74APfF(_TgduQnWmHF8RJhh28utaV6JTE5I , String[] )
   at _lt3zVdT0XPty8FTDA7a5BBWhtes._wGFDpKw5npWpK2EoZcBqD74APfF(ThreadStart )
-------------------------------

Event Log entries containing Vintagestory.exe, the latest 3
==================================
{ TimeGenerated = 13/07/2021 15:56:54, Site = , Source = .NET Runtime, Message = Application: Vintagestory.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0020001, exception address 00007FFCF16C4ED9
Stack:
 }```

**Comment by murilorodelli** [comment]
![image](https://user-images.githubusercontent.com/2029666/125636524-e100520f-0f64-4835-a2c0-c84102d360d4.png)

**Comment by murilorodelli** [comment]
I didn't mean to close, only upload a picture

**Comment by Craluminum2413** [comment]
I reproduced this

https://user-images.githubusercontent.com/69315569/125640782-c11ad1f9-86d0-4f14-8295-0c79b235cdcb.mp4

**Comment by murilorodelli** [comment]
[world.zip](https://github.com/anegostudios/VintageStory-Issues/files/6817119/world.zip)
[modpack.zip](https://github.com/anegostudios/VintageStory-Issues/files/6817125/modpack.zip)
I've attached the world file and the modpack..

**Comment by Craluminum2413** [comment]
They aren't needed since I reproduced this in single player without mods

**Comment by murilorodelli** [comment]
oh.. I see.

**Comment by radfast** [comment]
Thanks @murilorodelli for the careful report, and @Craluminum2413 for extra testing.  We'll fix this right away.

**Comment by Criperum** [comment]
Any news on this bug? It literally created a death zone around my base on TOTPS that noone can approach

**Comment by tyronx** [comment]
I can't reproduce this when using the instructions by @Craluminum2413, so perhaps its already fixed?

@Criperum what are your coords on TOPTS?

**Comment by Craluminum2413** [comment]
You need to automate pulverizer first

**Comment by tyronx** [comment]
thanks, that helped. I didnt see you place the pounder

**Comment by tyronx** [comment]
ok, thanks for the report, this will be fixed in 1.15.3
